packages/fast-component-explorer/app/utilities/configs/neutral-button.ts

```diff
@@ -0,0 +1,62 @@
+import { ComponentViewConfig } from "./data.props";
+import {
+    NeutralButton,
+    NeutralButtonProps,
+    neutralButtonSchema,
+} from "@microsoft/fast-components-react-msft";
+import Guidance from "../../.tmp/accent-button/guidance";
+import { glyphSchema, Icon } from "../../../app/components/glyph";
+
+const neutralButtonConfig: ComponentViewConfig<NeutralButtonProps> = {
+    schema: neutralButtonSchema,
+    component: NeutralButton,
+    guidance: Guidance,
+    scenarios: [
+        {
+            displayName: "Default",
+            data: {
+                children: "Neutral button",
+            },
+        },
+        {
+            displayName: "Anchor",
+            data: {
+                href: "#",
+                children: "Anchor neutral button",
+            },
+        },
+        {
+            displayName: "Disabled",
+            data: {
+                disabled: true,
+                children: "Disabled neutral button",
+            },
+        },
+        {
+            displayName: "Before content neutral button",
```

Comment:
same issue here, would rather have "Glyph before text" and below "Glyph after text"